docs/modules/ROOT/pages/how-to-guides/Import-code/proc_importing_code.adoc

```diff
@@ -226,8 +226,8 @@ Create the Red Hat Container Registry token by following these steps:
 . Create a registry service account by clicking *New Service Account*. 
```

Comment:
Create a registry service account by clicking *New Service Account*. You are taken to the page, **Create a New Registry Service Account**.